EpiAware/docs/src/man/getting-started-julia.md

```diff
@@ -0,0 +1,199 @@
+
+### Check versions of Julia installed
+
+To see a list of all the versions of Julia installed on your system, use the following command:
+
+```bash
+% juliaup list
+```
+
+### Update Julia (all versions installed)
+This will update all versions of Julia installed on your system to their latest release versions.
+
+```bash
+% juliaup update
+```
+
+##  Basic usage for Julia environments
+
+The [_environment_](https://docs.julialang.org/en/v1/manual/code-loading/#Environments-1) of a Julia project determines which packages, and their version, are available to the project. This is useful when you want to ensure that a project uses a specific version of a package, or when you want to isolate the project from other projects on your system. As per other languages, Julia environments are useful for managing dependencies and ensuring reproducibility.
+
+The most common usage of environments is to create a new [_explicit_ environment](https://docs.julialang.org/en/v1/manual/code-loading/#Environments) for a project in a directory. This creates a `Project.toml` file in the directory that specifies the dependencies for the project and a `Manifest.toml` file that specifies the exact versions of the dependencies, and their underlying dependencies. We'll discuss how to set up a new environment for a project in the [REPL section](#using-the-julia-repl).
+
+Julia environments can be [stacked](https://docs.julialang.org/en/v1/manual/code-loading/#Environment-stacks). This means that you can have a _primary_ environment that is used by default, and then _secondary_ environments that are used when you activate them. This is useful when you want common packages to be available to all projects, but also want to isolate projects from each other.
+
+Most commonly the environment for a Julia installation is the primary environment, and you can add packages to this environment that you want to be available to all projects as we'll show in the [REPL section](#using-the-julia-repl).
+
+## Using the Julia REPL in projects
+
+The Julia REPL (Read-Eval-Print Loop) is an interactive programming environment that takes single user inputs (i.e., single expressions), evaluates them, and returns the result to the user.
+
+### Package management programmatically and from REPL
+
+Julia has a built-in package manager called `Pkg`, which is documented briefly [here](https://docs.julialang.org/en/v1/stdlib/Pkg/) and in more detail [here](https://pkgdocs.julialang.org/v1/). The package manager is used to install, update, and manage Julia packages and environments.
+
+You can use `Pkg` programmatically as a normal Julia package, which is often done in scripts. For example, if we wanted to install the `OrdinaryDiffEq` package as part of executing a julia script, we would add the following lines to the script:
+
+```julia
+using Pkg
+Pkg.add("OrdinaryDiffEq")
+```
```

Comment:
Is this best practice? I can't tell is this is just an example of installing a package or if it's common to install packages as part of script-based workflows?

Reply:
Its quite contextual; there are situations where either is appropriate.

I'm trying to suggest that its possible whilst showing the way (I think) you should work ~80% of the time.

Reply:
Clarified this